fedora/36/Dockerfile

```diff
@@ -5,6 +5,9 @@ MAINTAINER Yaroslav Lobankov <y.lobankov@tarantool.org>
 # Fix missing locales
 ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
 
+# Replace repositories because fedora 36 reached EOL at 16 May 2023.
+COPY fedora.repo /etc/yum.repos.d/
```

Comment:
First, the official repos work now.

Second, if `fedora` repository will be removed, I suspect that `fedora-cisco-openh264`, `fedora-modular`, `updates-modular`, `updates` will be removed too. All of them are enabled now, see the build log below.

```
$ docker build --no-cache ./fedora/36 -t packpack/packpack:fedora-36-amd64-tmp
<...>
repo: downloading from remote: fedora
countme: no event for fedora: budget to spend: 3
Fedora 36 - x86_64                              2.6 MB/s |  81 MB     00:31    
fedora: using metadata from Wed May  4 21:16:11 2022.
repo: downloading from remote: fedora-cisco-openh264
Fedora 36 openh264 (From Cisco) - x86_64        1.1 kB/s | 2.5 kB     00:02    
fedora-cisco-openh264: using metadata from Thu Oct  6 11:02:51 2022.
repo: downloading from remote: fedora-modular
countme: no event for fedora-modular: budget to spend: 2
Fedora Modular 36 - x86_64                      2.2 MB/s | 2.4 MB     00:01    
fedora-modular: using metadata from Wed May  4 21:12:01 2022.
repo: downloading from remote: updates
countme: no event for updates: budget to spend: 2
Fedora 36 - x86_64 - Updates                    4.4 MB/s |  41 MB     00:09    
updates: using metadata from Tue May 16 02:57:55 2023.
repo: downloading from remote: updates-modular
countme: no event for updates-modular: budget to spend: 1
Fedora Modular 36 - x86_64 - Updates            2.1 MB/s | 2.9 MB     00:01    
updates-modular: using metadata from Mon Apr 10 00:16:43 2023.
```

Next, the same problem was fixed in a different way for fedora 35, see 7a465bdaffdace4f6b8c0706c78da9434ec523d2 (and you're the author of the commit). Can we reuse the same approach? It should handle the problem for all the repositories at once, not just the `fedora` repository.

Reply:
Ok. I see the problem of my commit. I fixed patch in same way https://github.com/packpack/packpack-docker-images/commit/7a465bdaffdace4f6b8c0706c78da9434ec523d2